puppet/modules/jenkins_node/manifests/init.pp

```diff
@@ -71,14 +71,14 @@
     ensure => file,
     owner  => 'jenkins',
     group  => 'jenkins',
-    source => 'puppet:///modules/slave/gitconfig',
+    source => 'puppet:///modules/jenkins_node/gitconfig',
```

Comment:
I guess this is why in the past many people used `${module_name}` instead of hardcoding it. Sometimes people do rename a module.

Reply:
Wait, `${module_name}` is a thing?! TIL

Reply:
https://www.puppet.com/docs/puppet/7/lang_facts_builtin_variables#lang_facts_builtin_variables-compiler-variables